pages/common/pdfjoin.md

```diff
@@ -0,0 +1,15 @@
+# pdfjoin
+
+> PDF merging utility.
+
+- Merge two PDFs:
+
+`pdfjoin {{file1}} {{file2}} {{output_file}}`
```

Comment:
If this is the `pdfjam` component, I think this needs to use `--outfile {{path/to/output}}`?

Reply:
Oops, you're right, I missed the switch. It indeed is part of `pdfjam`.